modules/katello_content_view.py

```diff
@@ -80,6 +81,9 @@
 except:
     HAS_NAILGUN_PACKAGE = False
 
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.foreman_helpers import handle_no_nailgun
```

Comment:
s/foreman_helpers/foreman_helper/

Reply:
You really seem to have a testing environment for all that stuff...
And i get lazy after doing the same change in about 6 different files.

Reply:
That one's just vim and the_silver_searcher :wink:

Reply:
But, we should *definitely* make a test suite.